### PR TITLE
fix: Default trame visualization in Jupyter environments

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
The default visualization breaks the usage of the plotter in Jupyter environments. When the documentation implementation is ready from PyVista side we should revisit how to generate the interactive plotters, since from now on, we will need "html" for documentation and ideally "trame" for regular Jupyter usage.

This arises from #897

### CODE REVIEW

**Comment by RobPasMue** [comment]
Doc build failing - have a look at it. I haven't read the logs yet.

File: doc/source/conf.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 from ansys.geometry.core import __version__
 
+os.environ["DOCUMENTATION_BUILD"] = "true"
```
Comment (RobPasMue) [inline doc/source/conf.py:22]:
```suggestion
os.environ["PYANSYS_GEOMETRY_DOC_BUILD"] = "true"
```

File: src/ansys/geometry/core/__init__.py [hunk 2]
```diff
@@ -60,3 +62,7 @@
 Only set this to false if you are sure you want to disable this check and you will ONLY
 be working with one design.
 """
+
+DOCUMENTATION_BUILD: bool = os.environ.get("PYGEOMETRY_DOC_BUILD", "false").lower() == "true"
```
Comment (RobPasMue) [inline src/ansys/geometry/core/__init__.py:66]:
```suggestion
DOCUMENTATION_BUILD: bool = os.environ.get("PYANSYS_GEOMETRY_DOC_BUILD", "false").lower() == "true"
```

**Review by RobPasMue — DISMISSED** [review]

**Review by RobPasMue — APPROVED** [review]

**Review by RobPasMue — APPROVED** [review]

**Review by RobPasMue — CHANGES_REQUESTED** [review]
Why this?

```os.environ["PYDEVD_DISABLE_FILE_VALIDATION"] = "1"```

**Comment by AlejandroFernandezLuces** [comment]
> Why this?
> 
> `os.environ["PYDEVD_DISABLE_FILE_VALIDATION"] = "1"`

Documentation building is getting frozen, and it throws this warning:
```
0.00s - Debugger warning: It seems that frozen modules are being used, which may
0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
0.00s - to python to disable frozen modules.
0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
```

I though I might give it a try

**Comment by RobPasMue** [comment]
> > Why this?
> > `os.environ["PYDEVD_DISABLE_FILE_VALIDATION"] = "1"`
> 
> Documentation building is getting frozen, and it throws this warning:
> 
> ```
> 0.00s - Debugger warning: It seems that frozen modules are being used, which may
> 0.00s - make the debugger miss breakpoints. Please pass -Xfrozen_modules=off
> 0.00s - to python to disable frozen modules.
> 0.00s - Note: Debugging will proceed. Set PYDEVD_DISABLE_FILE_VALIDATION=1 to disable this validation.
> ```
> 
> I though I might give it a try

Not remotely... I can't see that log on the workflows.

**Comment by AlejandroFernandezLuces** [comment]
Documentation looks good now. This should be good to go @RobPasMue

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**